README.md

```diff
@@ -1 +1 @@
-# Test Automation Samples
\ No newline at end of file
+# Test Automation Samples.
```

Comment:
@lmalakhova Need change README file by request of @sskorol in homework.
It should describe project structure and main features.